doc/sphinx/changes.rst

```diff
@@ -14,6 +14,152 @@ Version 9.0
+are also available.
+
+Théo Zimmermann with help from Ali Caglayan and Jason Gross maintained
+`coqbot <https://github.com/coq/bot>`_ used to run Coq's CI and other
+pull request management tasks.
+
+Jason Gross maintained the `bug minimizer <https://github.com/JasonGross/coq-tools>`_
+and its `automatic use through coqbot <https://github.com/coq/coq/wiki/Coqbot-minimize-feature>`_.
+
+Erik Martin-Dorel maintained the
+`Coq Docker images <https://hub.docker.com/r/coqorg/coq>`_
+and the `docker-keeper <https://gitlab.com/erikmd/docker-keeper>`_ compiler
+used to build and keep those images up to date (note that the tool is not Rocq specific).
+Cyril Cohen, Vincent Laporte, Pierre Roux and Théo Zimmermann
+maintained the `Nix toolbox <https://github.com/coq-community/coq-nix-toolbox>`_
+used by many Rocq projects for continuous integration.
+
+Ali Caglayan, Emilio Jesús Gallego Arias, Rudi Grinberg and
+Rodolphe Lepigre maintained the
+`Dune build system for OCaml and Coq/Rocq <https://github.com/ocaml/dune/>`_
+used to build Rocq itself and many Rocq projects.
+
+The `opam repository <https://github.com/coq/opam>`_ for Rocq packages has been maintained by
+Guillaume Claret, Guillaume Melquiond, Karl Palmskog, Matthieu Sozeau
+and Enrico Tassi with contributions from many users. The up-to-data list
+of packages is `available on the Rocq website <https://rocq-prover.org/packages>`_.
+
+Rocq 9.0 was made possible thanks to the following reviewers:
+TODO
+
+See the `Rocq Team <https://rocq-prover.org/governance>`_ page for
+more details on Rocq's development teams.
+
+The ?? contributors to the 9.0 version are:
+TODO
+
+The Coq/Rocq community at large helped improve this new version via
+the GitHub issue and pull request system, the coq-club@inria.fr mailing list,
+the `Discourse forum <https://coq.discourse.group/>`_ and the
+`Coq Zulip chat <https://coq.zulipchat.com>`_.
+
+Version 9.0's development spanned 7 months from the release of Coq 8.20.0.
+Pierre-Marie Pédrot and Matthieu Sozeau are the release managers of Rocq 9.0.
+This release is the result of ?? merged PRs, closing ?? issues.
+
+| Nantes, January 2025
+| Pierre-Marie Pédrot and Matthieu Sozeau for the Rocq development team
+
+Porting to The Rocq Prover
+--------------------------
+
+The Rocq Prover version 9.0 includes support for compability shims that allow
+to call it through legacy Coq commands (`coqtop`, `coqdep`, `coq_makefile`, `coqdoc`, TODO ...),
+so that it can act as a regular `Coq` version. If using `opam`, this compatibility
+layer is available through packages `coq-core`, `coq-stdlib` and `coq`. In this setting,
+nothing needs to be changed to an existing project to compile with `Rocq 9.0` (aliased as `Coq 9.0`).
+
+You should expect warnings that the standard library previously under namespace `Coq` has been renamed to
+`Stdlib` and moved to a `separate repositiory <https://github.com/coq/stdlib>`_, but these warnings are
+independent of the `Coq` to `Rocq` renaming.
```

Comment:
Maybe add a link to the precise changelog below that explain how to handle/silence the warning?

Reply:
Done